datagrepper/app.py

```diff
@@ -65,37 +64,6 @@
 import datagrepper.widgets  # noqa: E402,F401
 
 
-@app.context_processor
```

Comment:
Many of these variables injected here appear to still be referenced in the templates, e.g:

https://github.com/fedora-infra/datagrepper/blob/d1bb70f46285f1a34a8008c38b421c0606929bdd/datagrepper/templates/index.html#L75

do we need to remove this?